tests-functional/tests/test_status_connector.py

```diff
@@ -0,0 +1,66 @@
+import json
+import time
+import pytest
+
+from clients.connector import ConnectorClient
+from clients.signals import SignalType
+
+
+@pytest.mark.rpc
+@pytest.mark.connector
+class TestStatusConnector:
+
+    await_signals = [
+        SignalType.NODE_LOGIN.value,
+        SignalType.CONNECTOR_SEND_REQUEST_ACCOUNTS.value,
+        SignalType.CONNECTOR_SEND_TRANSACTION.value,
+        SignalType.CONNECTOR_SIGN.value,
+        SignalType.CONNECTOR_DAPP_PERMISSION_GRANTED.value,
+        SignalType.CONNECTOR_DAPP_PERMISSION_REVOKED.value,
+        SignalType.CONNECTOR_DAPP_CHAIN_ID_SWITCHED.value,
+    ]
+
+    @pytest.fixture
+    def backend(self, backend_new_profile):
+        yield backend_new_profile("sender", connector_enabled=True)
+
+    @pytest.fixture
+    def connector(self, backend):
+        time.sleep(5)
```

Comment:
hmm is this waiting for the service to init? can we do some healthcheck instead and assert if cannot connect in 5 seconds?

Reply:
oops, this is just a leftover, will remove, thanks!